[2024-12-06T19:13:16.075+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:13:15.012891+00:00 [queued]>
[2024-12-06T19:13:16.092+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:13:15.012891+00:00 [queued]>
[2024-12-06T19:13:16.093+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-06T19:13:16.107+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-12-06 19:13:15.012891+00:00
[2024-12-06T19:13:16.112+0000] {standard_task_runner.py:60} INFO - Started process 1252 to run task
[2024-12-06T19:13:16.114+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-12-06T19:13:15.012891+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpkfnfd5ko']
[2024-12-06T19:13:16.115+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask is_weather_api_ready
[2024-12-06T19:13:16.126+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-06T19:13:16.178+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:13:15.012891+00:00 [running]> on host 8a5e48d7ee4d
[2024-12-06T19:13:16.285+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T19:13:15.012891+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T19:13:15.012891+00:00'
[2024-12-06T19:13:16.286+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:13:16.295+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:13:20.236+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:13:20.236+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:14:20.280+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:14:20.289+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:14:21.702+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:14:21.703+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:15:21.747+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:15:21.759+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:15:24.199+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:15:24.199+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:16:23.569+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:16:23.582+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:16:24.857+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:16:24.857+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:17:24.919+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:17:24.925+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:17:26.069+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:17:26.070+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:18:26.099+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:18:26.107+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:18:30.898+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:18:30.898+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:19:30.960+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:19:30.975+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:19:34.110+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:19:34.111+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:20:34.172+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:20:34.186+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:20:34.401+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:20:34.402+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:21:34.462+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:21:34.472+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:21:35.129+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:21:35.130+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:22:35.190+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:22:35.203+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:22:35.714+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:22:35.715+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:23:35.726+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:23:35.737+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:23:41.426+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:23:41.426+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:24:41.454+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:24:41.470+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:24:42.166+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:24:42.167+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:25:42.187+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:25:42.198+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:25:45.642+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:25:45.642+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:26:45.662+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:26:45.668+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:26:46.658+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:26:46.659+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:27:46.720+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:27:46.732+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:27:51.538+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:27:51.538+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:28:51.566+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:28:51.576+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:28:54.213+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:28:54.214+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:29:54.277+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:29:54.286+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:29:54.501+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:29:54.502+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:30:54.544+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:30:54.556+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:30:54.959+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:30:54.960+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:31:55.016+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:31:55.029+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:31:55.267+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:31:55.268+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:32:55.279+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:32:55.291+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:32:57.063+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:32:57.063+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:33:57.124+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:33:57.137+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:33:58.983+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:33:58.984+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:34:59.045+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:34:59.051+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:35:05.523+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:35:05.524+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:36:05.585+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:36:05.597+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:36:09.162+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:36:09.163+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:37:09.225+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:37:09.238+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:37:13.395+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:37:13.396+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:38:13.459+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:38:13.471+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:38:16.391+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:38:16.392+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:39:15.933+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:39:15.944+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:39:17.607+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:39:17.608+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:40:17.671+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:40:17.683+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:40:18.556+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:40:18.557+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:41:18.620+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:41:18.630+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:41:19.516+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:41:19.517+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:42:19.579+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:42:19.592+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:42:20.997+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:42:20.998+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:43:21.061+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:43:21.067+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:43:23.017+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:43:23.017+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:44:23.040+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:44:23.053+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:44:25.236+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:44:25.236+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:45:25.298+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:45:25.304+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:45:26.833+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:45:26.834+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:46:26.883+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:46:26.889+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:46:27.759+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:46:27.760+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:47:27.822+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:47:27.833+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:47:31.075+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:47:31.076+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:48:31.136+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:48:31.149+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:48:36.508+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:48:36.509+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:49:36.571+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:49:36.583+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:49:39.013+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:49:39.014+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:50:39.030+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:50:39.037+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:50:41.027+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:50:41.028+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:51:41.074+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:51:41.087+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:51:45.847+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:51:45.848+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:52:45.911+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:52:45.923+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:52:49.028+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:52:49.029+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:53:49.090+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:53:49.101+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:53:52.088+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:53:52.088+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:54:52.149+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:54:52.155+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:54:52.879+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:54:52.879+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:55:52.941+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:55:52.953+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:55:55.011+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:55:55.011+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:56:55.060+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:56:55.066+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:56:58.179+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:56:58.180+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:57:58.241+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:57:58.253+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:58:04.995+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:58:04.996+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:59:05.031+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:59:05.043+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:59:09.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:59:09.061+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:00:09.122+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:00:09.129+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:00:11.451+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:00:11.452+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:01:11.513+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:01:11.524+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:01:13.028+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:01:13.029+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:02:13.092+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:02:13.106+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:02:15.043+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:02:15.043+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:03:15.103+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:03:15.109+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:03:17.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:03:17.061+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:04:17.088+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:04:17.100+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:04:17.805+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:04:17.805+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:05:17.826+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:05:17.838+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:05:19.563+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:05:19.564+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:06:19.626+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:06:19.633+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:06:20.284+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:06:20.285+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:07:20.345+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:07:20.356+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:07:25.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:07:25.061+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:08:25.123+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:08:25.128+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:08:27.076+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:08:27.077+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:09:27.140+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:09:27.147+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:09:27.940+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:09:27.941+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:10:28.003+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:10:28.008+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:10:30.366+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:10:30.366+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:11:29.861+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:11:29.875+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:11:30.553+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:11:30.554+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:12:30.616+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:12:30.626+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:12:33.850+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:12:33.851+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:13:33.913+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:13:33.920+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:13:38.553+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:13:38.554+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:14:39.085+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:14:39.096+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:14:40.972+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:14:40.973+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:15:41.036+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:15:41.041+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:15:41.803+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:15:41.804+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:16:41.865+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:16:41.876+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:16:45.004+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:16:45.004+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:17:45.066+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:17:45.078+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:17:49.256+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:17:49.256+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:18:49.318+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:18:49.329+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:18:52.708+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:18:52.708+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:19:52.770+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:19:52.777+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:19:54.444+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:19:54.444+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:20:54.506+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:20:54.511+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:20:56.657+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:20:56.658+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:21:56.720+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:21:56.733+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:21:57.451+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:21:57.451+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:22:57.513+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:22:57.525+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:23:00.684+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:23:00.685+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:24:00.748+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:24:00.759+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:24:02.651+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:24:02.651+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:25:02.712+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:25:02.718+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:25:04.972+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:25:04.972+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:26:05.033+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:26:05.044+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:26:05.789+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:26:05.790+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:27:05.852+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:27:05.865+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:27:07.681+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:27:07.682+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:28:07.744+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:28:07.756+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:28:09.049+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:28:09.050+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:29:09.081+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:29:09.085+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:29:10.800+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:29:10.801+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:30:10.863+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:30:10.874+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:30:12.337+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:30:12.338+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:31:12.400+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:31:12.406+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:31:14.955+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:31:14.955+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:32:15.016+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:32:15.024+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:32:17.612+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:32:17.612+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:33:17.674+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:33:17.680+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:33:24.989+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:33:24.989+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:34:25.031+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:34:25.043+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:34:25.268+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:34:25.269+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:35:25.331+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:35:25.338+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:35:28.972+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:35:28.973+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:36:29.036+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:36:29.045+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:36:33.035+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:36:33.035+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:37:33.064+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:37:33.071+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:37:37.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:37:37.045+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:38:37.064+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:38:37.079+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:38:40.971+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:38:40.971+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:39:41.032+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:39:41.044+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:39:46.327+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:39:46.327+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:40:46.388+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:40:46.396+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:40:46.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:40:46.987+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:41:46.994+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:41:47.002+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:41:47.203+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:41:47.203+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:42:47.265+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:42:47.280+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:42:49.484+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:42:49.484+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:43:49.547+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:43:49.556+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:43:50.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:43:50.061+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:44:50.124+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:44:50.128+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:44:51.108+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:44:51.109+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:45:51.173+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:45:51.180+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:45:51.492+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:45:51.493+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:46:51.545+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:46:51.556+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:46:53.013+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:46:53.013+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:47:53.049+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:47:53.053+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:47:53.853+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:47:53.853+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:48:53.914+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:48:53.924+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:48:56.231+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:48:56.231+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:49:56.293+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:49:56.304+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:49:58.604+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:49:58.605+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:50:58.615+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:50:58.624+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:50:59.085+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:50:59.086+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:51:59.148+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:51:59.155+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:51:59.651+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:51:59.651+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:52:59.684+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:52:59.697+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:52:59.924+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:52:59.925+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:53:59.987+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:53:59.992+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:54:00.800+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:54:00.801+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:55:00.815+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:55:00.827+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:55:01.643+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:55:01.644+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:56:01.705+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:56:01.718+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:56:02.308+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:56:02.309+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:57:02.371+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:57:02.377+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:57:02.552+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:57:02.552+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:58:02.593+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:58:02.598+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:58:04.235+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:58:04.235+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:59:04.294+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:59:04.308+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:59:05.270+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:59:05.271+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:00:05.334+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:00:05.343+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:00:07.372+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:00:07.373+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:01:07.435+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:01:07.441+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:01:08.548+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:01:08.549+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:02:08.610+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:02:08.616+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:02:09.047+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:02:09.047+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:03:09.054+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:03:09.066+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:03:09.899+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:03:09.899+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:04:09.961+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:04:09.973+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:04:11.020+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:04:11.021+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:05:11.082+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:05:11.089+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:05:13.485+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:05:13.485+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:06:13.544+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:06:13.550+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:06:14.868+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:06:14.869+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:07:14.931+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:07:14.941+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:07:16.333+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:07:16.334+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:08:16.396+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:08:16.408+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:08:21.003+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:08:21.004+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:09:21.066+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:09:21.071+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:09:22.955+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:09:22.955+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:10:22.982+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:10:22.993+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:10:26.012+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:10:26.012+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:11:26.074+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:11:26.080+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:11:26.830+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:11:26.831+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:12:26.849+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:12:26.856+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:12:27.972+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:12:27.973+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:13:28.019+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:13:28.031+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:13:32.930+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:13:32.930+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:14:32.993+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:14:33.001+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:14:37.002+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:14:37.002+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:15:37.063+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:15:37.070+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:15:40.812+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:15:40.813+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:16:40.827+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:16:40.839+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:16:41.969+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:16:41.969+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:17:42.031+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:17:42.040+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:17:42.854+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:17:42.855+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:18:42.915+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:18:42.926+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:18:44.427+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:18:44.427+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:19:44.488+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:19:44.502+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:19:44.675+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:19:44.675+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:20:44.694+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:20:44.700+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:20:45.172+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:20:45.173+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:21:45.235+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:21:45.245+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:21:46.955+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:21:46.956+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:22:47.017+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:22:47.028+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:22:47.426+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:22:47.426+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:23:47.487+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:23:47.492+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:23:49.643+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:23:49.643+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:24:49.704+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:24:49.715+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:24:49.912+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:24:49.912+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:25:49.962+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:25:49.975+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:25:52.677+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:25:52.678+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:26:52.740+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:26:52.745+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:26:53.043+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:26:53.043+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:27:53.057+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:27:53.067+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:27:55.275+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:27:55.275+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:28:55.325+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:28:55.335+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:28:56.781+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:28:56.782+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:29:57.330+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:29:57.337+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:29:57.673+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:29:57.674+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:30:57.734+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:30:57.740+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:30:58.287+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:30:58.288+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:31:58.318+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:31:58.330+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:31:59.508+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:31:59.508+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:32:59.567+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:32:59.578+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:33:01.075+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:33:01.076+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:34:01.137+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:34:01.150+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:34:02.645+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:34:02.645+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:35:02.707+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:35:02.720+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:35:03.509+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:35:03.510+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:36:03.572+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:36:03.581+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:36:04.276+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:36:04.276+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:37:04.337+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:37:04.345+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:37:04.685+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:37:04.686+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:38:04.748+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:38:04.755+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:38:05.962+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:38:05.963+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:39:05.577+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:39:05.583+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:39:07.094+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:39:07.094+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:40:07.155+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:40:07.166+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:40:07.716+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:40:07.717+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:41:07.749+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:41:07.763+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:41:09.921+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:41:09.921+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:42:09.975+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:42:09.985+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:42:14.083+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:42:14.083+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:43:14.145+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:43:14.153+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:43:16.046+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:43:16.047+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:44:16.099+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:44:16.107+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:44:17.684+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:44:17.684+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:45:17.745+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:45:17.757+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:45:18.655+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:45:18.655+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:46:18.716+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:46:18.725+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:46:19.924+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:46:19.924+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:47:19.985+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:47:19.992+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:47:20.236+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:47:20.237+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:48:20.240+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:48:20.245+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:48:23.574+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:48:23.574+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:49:23.637+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:49:23.646+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:49:24.836+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:49:24.836+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:50:24.897+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:50:24.907+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:50:25.115+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:50:25.115+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:51:25.162+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:51:25.170+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:51:26.548+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:51:26.548+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:52:26.609+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:52:26.616+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:52:27.093+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:52:27.094+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:53:27.097+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:53:27.108+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:53:27.700+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:53:27.701+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:54:27.721+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:54:27.728+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:54:28.500+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:54:28.501+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:55:28.530+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:55:28.536+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:55:29.381+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:55:29.381+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:56:29.417+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:56:29.424+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:56:31.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:56:31.046+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:57:31.080+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:57:31.093+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:57:31.788+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:57:31.788+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:58:31.849+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:58:31.856+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:58:35.028+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:58:35.029+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:59:35.090+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:59:35.100+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:59:35.373+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:59:35.374+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:00:35.435+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:00:35.441+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:00:37.076+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:00:37.076+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:01:37.138+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:01:37.149+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:01:37.774+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:01:37.775+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:02:37.836+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:02:37.843+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:02:39.614+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:02:39.615+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:03:39.676+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:03:39.683+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:03:42.934+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:03:42.935+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:04:42.996+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:04:43.013+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:04:47.062+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:04:47.063+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:05:47.074+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:05:47.080+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:05:47.557+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:05:47.557+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:06:47.618+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:06:47.626+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:06:47.886+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:06:47.887+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:07:47.950+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:07:47.960+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:07:48.902+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:07:48.903+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:08:48.965+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:08:48.972+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:08:49.193+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:08:49.194+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:09:49.255+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:09:49.261+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:09:51.316+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:09:51.317+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:10:51.379+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:10:51.394+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:10:51.628+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:10:51.629+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:11:51.667+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:11:51.672+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:11:51.865+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:11:51.866+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:12:51.928+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:12:51.939+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:12:52.172+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:12:52.173+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:13:52.233+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:13:52.241+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:13:52.493+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:13:52.494+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:14:52.555+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:14:52.562+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:14:52.772+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:14:52.772+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:15:52.833+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:15:52.839+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:15:53.508+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:15:53.509+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:16:53.569+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:16:53.579+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:16:55.360+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:16:55.360+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:17:55.422+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:17:55.436+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:17:56.528+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:17:56.529+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:18:56.591+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:18:56.603+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:19:00.117+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:19:00.117+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:00.179+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:20:00.184+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:20:02.900+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:20:02.900+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:53.067+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2024-12-06T22:20:53.068+0000] {process_utils.py:131} INFO - Sending 15 to group 1252. PIDs of all processes in the group: [1252]
[2024-12-06T22:20:53.068+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 1252
[2024-12-06T22:20:53.068+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-06T22:20:53.094+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1252, status='terminated', exitcode=0, started='19:13:15') (1252) terminated with exit code 0
